docs/dev/api_integration.md

```diff
@@ -17,149 +19,35 @@ API integrations send the following types of data to Datadog:
-
-Follow this template to define the `media` object in the media carousel which includes an image, a video thumbnail, and a video:
-
-{{< code-block lang="json" filename="manifest.json" collapsible="true" >}}
-"media": [
-      {
-        "media_type": "image",
-        "caption": "A Datadog API Integration OOTB Dashboard",
-        "image_url": "images/integration_name_image_name.png"
-      },
-      {
-        "media_type": "video",
-        "caption": "A Datadog API Integration Overview Video",
-        "image_url": "images/integration_name_video_thumbnail.png",
-        "vimeo_id": 123456789
-      },
-    ],
-{{< /code-block >}}
-
-For more information, see [Integrations Assets Reference][20].
-
-## Open a pull request
+- You must implement OAuth in order to submit data or pull data out of Datadog. 
+- You must support all [Datadog sites][12].
 
-Open a pull request in the [`integrations-extras` repository][21] that adds images (such as logos and images) and asset files (such as `Changelog.md`, `README.md`, and `manifest.json`) to your API integration's tile-only listing in the [Integrations page][22]. Automatic tests run checks to verify that your pull request is in good shape and contains all the required content to be updated.
+### OAuth
+Instead of requesting API and Application keys directly from a user, Datadog requires using an [OAuth client][14] to handle authorization and access for API-based integrations. For more information, see [OAuth for Integrations][15] and [Authorization Endpoints][16]. 
 
-## Review process
+You can explore examples that use OAuth in the `integrations-extras` repository such as [Vantage][17].
 
-Once your pull request passes all the checks, reviewers from the `Datadog/agent-integrations`, `Datadog/marketplace`, and `Datadog/documentation` teams provide suggestions and feedback on best practices.
+## Development Process
 
-Once you have addressed the feedback and re-requested reviews, these reviewers approve your pull request.
+You can expect the following process for building an API-based integration:
+1. Meet with the Datadog Technology Partner team to discuss your offering and use cases.
+2. Request a sandbox account for development.
+3. Begin development of your integration, which will include writing and hosting integration code on your end, as well as implementing the OAuth protocol.
+4. Test your integration, as well as your OAuth client, in your sandbox account.
+5. Once your development work is tested and complete, **follow the steps to [Create a Tile][24] in order to display your offering on the Marketplace or Integrations page**.
+6. Once your pull request is submitted and approved, the team will schedule a demo for a final review of your integration.
+7. You'll have the option of testing the tile and integration in your sandbox account before publishing, or immediately publishing the integration for all customers. 
 
+To get started building a tile, [click here][24]. 
```

Comment:
Just my two cents, but the bolding in number 5 with the redundant links are a tad confusing.